vspace/src/main/java/edu/asu/diging/vspace/core/services/impl/ContentBlockManager.java

```diff
@@ -133,7 +135,48 @@ public CreationReturnValue createImageBlock(String slideId, byte[] image, String
         return returnValue;
     }
 
+    /**
+     * Delete a text block using an id
+     * 
+     * @param id - id of resource to be deleted. If the id is null then the
+     *           functions returns nothing.
+     *
+     */
+
     @Override
+    public void deleteTextBlockById(String id) throws BlockDoesNotExistException {
+        if (id == null) {
+            return;
+        }
+        try {
+            textBlockRepo.deleteById(id);         
+        } catch (EmptyResultDataAccessException e) {
```

Comment:
Are you sure this exception is being thrown here?

Reply:
If someone was to edit the id of the text block in the console of browser, then it would throw the error. It can also be thrown if somehow data gets deleted from db while the user has the page opened and then user tries to delete it through UI. Highly unlikely scenarios, but can happen. This code was already present in the branch from prior commits by Taylor I believe, so I did not make changes to these. Removing it will stop showing a generic message on the UI that something went wrong in case something does go wrong.

Reply:
Based on the comments above, please let me know if I should remove EmptyResultDataAccessException or not.

Reply:
did you look into the javadoc? I don't think this exception is thrown by this method. (it's not a question of could it be thrown but does the method ever actually throw it based on its implementation).